components/server/src/user/user-controller.ts

```diff
@@ -511,6 +511,18 @@ export class UserController {
         await this.userService.updateUserEnvVarsOnLogin(user, envVars);
         await this.userService.acceptCurrentTerms(user);
         this.analytics.identify({ anonymousId: req.sessionID || "unknown", userId: user.id, });
+        this.analytics.track({ 
```

Comment:
please wrap both analytics calls in a `try-catch` to limit any side effects. 🙏🏻

Reply:
NVM, just learned, that it's already error-handled internally. just missed that change 🤷🏻‍♂️